src/Core/Tests/ResultTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\Core\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+class ResultTest extends TestCase
```

Comment:
It seams like this test is testing SimpleXmlElement. 

We could make a comment why we need this test here

Reply:
added a comment